tests/datasource/data_connector/test_data_connector_query.py

```diff
@@ -414,7 +411,7 @@ def my_custom_batch_selector(batch_identifiers: dict) -> bool:
             data_connector_name="general_filesystem_data_connector",
             data_asset_name="TestFiles",
             batch_identifiers=IDDict(
-                {"name": "james", "timestamp": "20200811", "price": "1009"}
+                {"name": "james", "timestamp": "20200810", "price": "1003"}
```

Comment:
we had configured DateTimeSorter as descending which means 20200810 would have been the `-1` index or last entry ( `james_20200713_1567.csv` would  have been filtered by the `my_custom_batch_selector()` above